geometry_msgs/TransformStamped right_controller_transform
geometry_msgs/TransformStamped left_controller_transform
bool right_controller_present
bool left_controller_present
bool A
bool B
bool X
bool Y
bool RThU  # Right thumb up
bool LThU  # Left thumb up
bool RJ  # Right joystick pressed
bool LJ  # Left joystick pressed
bool RG  # Right grip button pressed
bool LG  # Left grip button pressed
bool RTr  # Right trigger pressed
bool LTr  # Left trigger pressed
float32 right_joystick_x
float32 right_joystick_y
float32 left_joystick_x
float32 left_joystick_y
float32 right_grip
float32 left_grip
float32 right_trigger
float32 left_trigger